src/BitcoinHelpers.js

```diff
@@ -45,12 +58,25 @@ const BitcoinNetwork = {
  */
 
 /**
- * @typedef {Object} SPVProof
- * @extends {Proof}
- * @property {ParsedTransaction} parsedTransaction Parsed transaction with
+ * @typedef {object} SPVProofInfo
+ * @property {HexTransactionFields} parsedTransaction Parsed transaction with
  *           additional data useful in submitting SPV proofs, stored as buffers.
  */
 
+/**
+ * @typedef {Proof & SPVProofInfo} SPVProof
```

Comment:
The previous way didn't correctly articulate that `SPVProof` should have everything in `Proof` + the `parsedTransaction` field.